timesketch/app.py

```diff
@@ -192,6 +192,19 @@ def load_user(user_id):
     # Setup CSRF protection for the whole application
     CSRFProtect(app)
 
+    if app.config.get("DEBUG"):
+        # pylint: disable=import-outside-toplevel
+        from werkzeug.middleware.profiler import ProfilerMiddleware
+
+        # Profiles are stored in a 'profiles' directory in the project root.
+        # app.root_path is the path to the 'timesketch' package directory.
+        project_root = os.path.dirname(app.root_path)
+        profile_dir = os.path.join(project_root, "profiles")
```

Comment:
On a system using the latest or release images this folder will be `/usr/local/lib/python3.10/dist-packages/profiles/` and I don't think you want to drop the file there? 

I would recommend to write the files to `/var/log/timesketch/profiles/` instead since this is mapped into the host as well. 

Project root with `timesketch-dev` container:
```
>>> project_root = os.path.dirname(app.root_path)
>>> project_root
'/usr/local/src/timesketch'
```

So you should be able to detect if you are running in timesketch-dev this way.